src/components/Analytics.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import React from 'react'
+
+class Analytics extends React.Component {
```

Comment:
i'm not convinced this is better than what we have; to me, the componentizing (of this) adds misdirection that feels a little confusing/clunky